ADBench/plot_graphs.py

```diff
@@ -172,8 +162,40 @@ def div_lists(alist, blist):
 
 
 
-if do_help:
-    ref_msg = f'''
+def get_style(tool, tool_names):
+    '''Returns the style of the given tool from all tool list.'''
+
+    idx = tool_names.index(tool)
```

Comment:
But why not just hash the name?  This means new tools will change the colours.  And we had to sort etc.  The chance of collision is exactly the same.

Reply:
But this way we will never have collisions, except the count of tools is greater than count of styles, which can always be expanded if it necessary. Honestly, I think for this simple case hashing is too powerful solution in differ with simple tool name sorting. But it is only my opinion

Reply:
Oh yes, sorry.  Agreed - chance of collision is indeed high.

Reply:
So remove the sorting, and just use a fixed mapping toolname -> style as initially discussed.

Reply:
Maybe, just fix the legend? https://github.com/awf/ADBench/pull/127#discussion_r334376092

Reply:
> So remove the sorting, and just use a fixed mapping toolname -> style as initially discussed.

I think after my the last commit it is still unnecessary